omniledger/darc/struct.go

```diff
@@ -43,7 +43,16 @@ type Darc struct {
 	BaseID ID
 	// PrevID is the previous darc ID in the chain of evolution.
 	PrevID ID
-	// Rules map an action to an expression.
+	// EvolveName is the name for the evolution action, which must
+	// exist and is responsible for managing the rules for evolving a darc.
+	// This field should not be changed after initialisation.
```

Comment:
"should not change after init" -> Then why is it public?

Reply:
Because they need to be serialised by protobuf